app_legacy/Helpers/render/achievement.php

```diff
@@ -45,9 +48,30 @@ function achievementAvatar(
         iconSize: $iconSize,
         iconClass: $iconClass,
         context: $context,
+        sanitize: $title === null,
+        altText: $title ?? $label,
     );
 }
 
+/**
+ * Render achievement title, parsing `[m]` (missable) as a tag
+ */
+function renderAchievementTitle(string $title, bool $tags = true): string
+{
+    if (!str_contains($title, '[m]')) {
+        return $title;
+    }
+    $span = '';
+    if ($tags) {
+        $span = "<span class='tag missable' title='Missable'>"
+            . "<div>[<span>m</span>]</div>"
```

Comment:
As it's an inline element this should probably be a `span` instead of a `div`?

Reply:
Hmmm yeah, it seems it isn't a good convention putting `div`s inside `span`s. And the context for tags is inline since they flow alongside text. To be honest, I'm just not fond of having something like `<span><span>...<span>...</span>...</span></span>`. I'll see if I can find a semantically good way of addressing this.

Reply:
P.S: the reason we need an intermediate `<span>` is just because `.tag` is `inline-flex`, which breaks the inner text when searching with `Ctrl+F`.

Reply:
Okay, I think this feels better now:
```html
<span class='tag missable' title='Missable'>
   <abbr>[<b>m</b>]</abbr>
</span>
```